meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx8/boot.cmd

```diff
@@ -5,28 +5,46 @@ echo "Using freescale_${fdt_file}"
 # Default boot type and device
 setenv bootlimit 3
 setenv devtype mmc
-setenv devnum ${mmcdev}
+setenv devnum 0
+setenv bootpart 1
+setenv rootpart 2
 setenv hdmi_image hdmitxfw.bin
 setenv m4_0_image core0_m4_image.bin
 setenv m4_1_image core1_m4_image.bin
+setenv ramdisk_addr_r 0x8a000000
+# enable relocation of ramdisk
+setenv initrd_high
```

Comment:
IIRC enabling relocation caused some side-effects on Apalis iMX8 (I haven't debugged that problem though). Haven't you seen any Linux boot issues after?

Reply:
Dropped the re-define of ramdisk_addr_r and the relocation of ramdisk